akka-remote/src/test/scala-jdk9-only/akka/remote/artery/jfr/RemotingFlightRecorderSpec.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (C) 2009-2023 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.remote.artery.jfr
+
+import com.typesafe.config.ConfigFactory
+
+import akka.actor.ActorSystem
+import akka.remote.artery.RemotingFlightRecorder
+import akka.testkit.AkkaSpec
+import akka.testkit.TestKit
+
+class RemotingFlightRecorderSpec extends AkkaSpec {
```

Comment:
Can we remove this spec at all?

Reply:
Yeah, it only tests the toggle, same for both JFR specs